src/handycon/utilities.py

```diff
@@ -175,6 +176,12 @@ def id_system():
         handycon.system_type = "AYN_GEN3"
         ayn_gen3.init_handheld(handycon)
 
```

Comment:
Can you add a Lenovo section header to the comments in the same style as the other manufacturers please.

Reply:
done